File: Pidev_EGOV/EGOV_symfony/src/MyApp/UserBundle/Entity/Articles.php
<?php

namespace MyApp\UserBundle\Entity;
use Symfony\Component\Validator\Constraints as Assert;
use Doctrine\ORM\Mapping as ORM;
use FOS\CommentBundle\Model\Comment as BaseComment;
/**
 * Articles
 *
 * @ORM\Table(name="articles")
 * @ORM\Entity
 */
class Articles extends BaseComment
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id_article", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idArticle;

    /**
     * @var string
     *
     * @ORM\Column(name="titre_article", type="string", length=20, nullable=false)
     */
    private $titreArticle;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date_article", type="date", nullable=false)
     */
    private $dateArticle;

    /**
     * @var string
     *
     * @ORM\Column(name="image_article", type="blob", length=65535, nullable=true)
     */
    private $imageArticle;

    /**
     * @var string
     *
     * @ORM\Column(name="texte_article", type="text", length=65535, nullable=false)
     */
    private $texteArticle;

    /**
     * @var integer
     *
     * @ORM\Column(name="priorite_pub", type="integer", nullable=false)
     */
    private $prioritePub;
    
    /**
     * @var integer
     *
     * @ORM\Column(name="type", type="string", nullable=false)
     */
    private $type;
    /** 
     * 
     * @Assert\File(maxSize="6000000") 
     * 
     */
     private $file;

 

    /**
     * Get idArticle
     *
     * @return integer 
     */
    public function getIdArticle()
    {
        return $this->idArticle;
    }

    /**
     * Set titreArticle
     *
     * @param string $titreArticle
     * @return Articles
     */
    public function setTitreArticle($titreArticle)
    {
        $this->titreArticle = $titreArticle;

        return $this;
    }

    /**
     * Get titreArticle
     *
     * @return string 
     */
    public function getTitreArticle()
    {
        return $this->titreArticle;
    }

    /**
     * Set dateArticle
     *
     * @param \DateTime $dateArticle
     * @return Articles
     */
    public function setDateArticle($dateArticle)
    {
        $this->dateArticle = $dateArticle;

        return $this;
    }

    /**
     * Get dateArticle
     *
     * @return \DateTime 
     */
    public function getDateArticle()
    {
        return $this->dateArticle;
    }

    /**
     * Set imageArticle
     *
     * @param string $imageArticle
     * @return Articles
     */
    public function setImageArticle($imageArticle)
    {
        $this->imageArticle = $imageArticle;

        return $this;
    }

    /**
     * Get imageArticle
     *
     * @return string 
     */
    public function getImageArticle()
    {
        return $this->imageArticle;
    }

    /**
     * Set texteArticle
     *
     * @param string $texteArticle
     * @return Articles
     */
    public function setTexteArticle($texteArticle)
    {
        $this->texteArticle = $texteArticle;

        return $this;
    }

    /**
     * Get texteArticle
     *
     * @return string 
     */
    public function getTexteArticle()
    {
        return $this->texteArticle;
    }

    /**
     * Set prioritePub
     *
     * @param integer $prioritePub
     * @return Articles
     */
    public function setPrioritePub($prioritePub)
    {
        $this->prioritePub = $prioritePub;

        return $this;
    }

    /**
     * Get prioritePub
     *
     * @return integer 
     */
    public function getPrioritePub()
    {
        return $this->prioritePub;
    }
    
    /**
     * Set type
     *
     * @param string $type
     * @return Articles
     */
    public function setType($type)
    {
        $this->type = $type;

        return $this;
    }

    /**
     * Get type
     *
     * @return string 
     */
    public function getType()
    {
        return $this->type;
    }
     
    public function setFile($file)
    {
        $this->file = $file;
     
        return $this;
    }
 
 
    public function getFile()
    {
        return $this->file;
    }
   
    
}

File: Pidev_EGOV/piegovj2me/src/edu/esprit/piegov/entites/Articles.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package edu.esprit.piegov.entites;


import java.io.InputStream;
import java.util.Date;


/**
 *
 * @author USER
 */
public class Articles {
    
    private int id_article;
    private String titre_article;
    private Date date_article;
//    private Blob image_article;
    private String texte_article;
    private int priorite_pub;
    private String type;
    private InputStream fin ;

    public InputStream getFin() {
        return fin;
    }

    public void setFin(InputStream fin) {
        this.fin = fin;
    }
    public Articles(int id_article, String titre_article, Date date_article, String texte_article, int priorite_pub, String type) {
        this.id_article = id_article;
        this.titre_article = titre_article;
        this.date_article = date_article;
     //   this.image_article = image_article;
        this.texte_article = texte_article;
        this.priorite_pub = priorite_pub;
        this.type = type;
    }

    public Articles() {
    }

    public Articles(int id, String titre, Date sqlDatecreation, InputStream is, String texte, int priorite, String type) {
        this.id_article = id;
        this.titre_article = titre;
        this.date_article = sqlDatecreation;
        this.fin = is;
        this.texte_article = texte;
        this.priorite_pub = priorite;
        this.type = type;
       
    }

    public int getId_article() {
        return id_article;
    }

    public void setId_article(int id_article) {
        this.id_article = id_article;
    }

    public String getTitre_article() {
        return titre_article;
    }

    public void setTitre_article(String titre_article) {
        this.titre_article = titre_article;
    }

    public Date getDate_article() {
        return date_article;
    }

    public void setDate_article(Date date_article) {
        this.date_article = date_article;
    }

//    public Blob getImage_article() {
//        return image_article;
//    }

//    public void setImage_article(Blob image_article) {
//        this.image_article = image_article;
//    }

    public String getTexte_article() {
        return texte_article;
    }

    public void setTexte_article(String texte_article) {
        this.texte_article = texte_article;
    }

    public int getPriorite_pub() {
        return priorite_pub;
    }

    public void setPriorite_pub(int priorite_pub) {
        this.priorite_pub = priorite_pub;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

  
    public int hashCode() {
        int hash = 3;
        hash = 53 * hash + this.id_article;
        return hash;
    }

 

    public String toString() {
        return "Articles{" + "id_article=" + id_article + ", titre_article=" + titre_article + ", date_article=" + date_article +  ", texte_article=" + texte_article + ", priorite_pub=" + priorite_pub + ", type=" + type + '}';
    }
    
    
    
}
